editor-components/header/header26/header26.tsx

```diff
@@ -1,120 +1,281 @@
+      value: "IoIosArrowDown",
     });
   }
 
   getName(): string {
     return "Header-26";
   }
 
+  handleBeforeChange = () => {
+    const sliderClass = this.decorateCSS("sliders");
+    if (this.sliderRef.current) {
+      const slides = this.sliderRef.current.innerSlider.list.querySelectorAll(`.${sliderClass}`);
+      slides.forEach((slide: HTMLElement) => {
+        slide.classList.add(this.decorateCSS("shrink"));
+      });
+    }
+  };
+
+  handleAfterChange = () => {
+    const sliderClass = this.decorateCSS("sliders");
+    if (this.sliderRef.current) {
+      const slides = this.sliderRef.current.innerSlider.list.querySelectorAll(`.${sliderClass}`);
+      slides.forEach((slide: HTMLElement) => {
+        slide.classList.remove(this.decorateCSS("shrink"));
+      });
+    }
+  };
+
+
+  handlePrevNextChange = () => {
+    this.handleBeforeChange();
+    setTimeout(() => {
+      this.handleAfterChange();
+    }, 1500);
+  };
+
+  handlePrevClick = () => {
+    if (this.sliderRef.current) {
+      this.sliderRef.current.slickPrev();
+      this.handlePrevNextChange();
+    }
+  };
+
+  handleNextClick = () => {
+    if (this.sliderRef.current) {
+      this.sliderRef.current.slickNext();
+      this.handlePrevNextChange();
+    }
+  };
+
   render() {
-    let items: any = this.castToObject("items");
-    const button = this.castToObject<any>("button")    
+    const settings = {
+      dots: false,
+      arrows: false,
+      infinite: true,
+      speed: 1500,
+      autoplay: false,
+      autoplaySpeed: 3000,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+      swipeToSlide: true,
+      draggable: true,
+      vertical: true,
+      verticalSwiping: true,
+      beforeChange: this.handleBeforeChange,
+      afterChange: this.handleAfterChange,
+    };
 
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("wrapper")}>
-            <div className={this.decorateCSS("background-image")}>
-              <img src={items["left-top-item"].image} alt={items["left-top-item"].title} />
-            </div>
-            <div className={this.decorateCSS("left")}>
-              <div className={this.decorateCSS("title")}>
-                <h1 className={this.decorateCSS("items")}>{items["left-top-item"].title}</h1>
-              </div>
-              <div className={this.decorateCSS("description")}>
-                <h3 className={this.decorateCSS("items-description")}>{items["left-top-item"].description}</h3>
-              </div>
-
-              
-                  <ComposerLink path={button.url}>
-                    <div className={this.decorateCSS("button-wrapper")}>
-                      <a
-                        className={this.decorateCSS("button")}
+          <ComposerSlider {...settings} ref={this.sliderRef}>
+            {this.castToObject<[]>("sliders").map((item: any, index: number) => (
```

Comment:
after creating the relevant type, cast it to that.
e.g. `this.castToObject<SliderItem[]>("sliders").map((item: SliderItem, ...`